lib/kube/proxy/forwarder.go

```diff
@@ -66,6 +66,10 @@ import (
 	utilexec "k8s.io/client-go/util/exec"
 )
 
+func init() {
+	mathrand.Seed(time.Now().UnixNano())
```

Comment:
What do you think about moving this to `main` instead? It really only needs to be called once, and if we start to do it in a package's `init()` function then we could end up with multiple packages making the same call.

Reply:
Hmm In that case it wouldn't be seeded in tests, or any other package outside of `tools/teleport` that might import it. If anything it might be better to move this to a `lib/utils/rand` package and reuse it to avoid this issue. What do you think?

Reply:
You don't really need to seed it for tests. It's fine if you get the same ordering every time in tests. But if you really wanted to you could put it in a `TestMain`.

Reply:
I also think it'd be most appropriate to initialize the generator globally by moving this call to `main`. If you wanted to have a scope-limited generator for some reason, consider using a `rand.Rand` instance.